balancer/weightedtarget/weightedtarget_test.go

```diff
@@ -58,6 +63,17 @@ func newTestConfigBalancerBuilder() *testConfigBalancerBuilder {
 	}
 }
 
+func makeRPCsAndAssertContainsError(rpcCount int, want error) func(balancer.Picker) error {
+	return func(p balancer.Picker) error {
+		for i := 0; i < rpcCount; i++ {
+			if _, err := p.Pick(balancer.PickInfo{}); !strings.Contains(err.Error(), want.Error()) {
+				return fmt.Errorf("want pick error to contain %q, got %q", want, err)
```

Comment:
Got before want. See: go/go-style/decisions#got-before-want

Reply:
i think i copied a wrong error from before. Let me correct that